Gemfile

```diff
@@ -15,6 +15,8 @@ gem "decidim-notify", "~> 0.5"
 gem "decidim-templates", DECIDIM_VERSION
 gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer", branch: :develop
 
+gem "decidim-multisurveys", path: "./decidim-multisurveys"
```

Comment:
I'm creating this component as an internal module so it's encapsulated and can be later on used in other projects if necessary.